actix-web-location/src/extractors.rs

```diff
@@ -28,21 +28,21 @@ impl FromRequest for Location {
                         if let Some(metrics) = config.metrics.as_ref() {
                             if provider.expect_city() && location.city.is_none() {
                                 metrics
-                                    .incr_with_tags("location.unknown.city")
+                                    .count_with_tags("location.unknown.city", 1)
```

Comment:
The incr methods are still available but now via `CountedExt` in .26

Reply:
ah, that's where they went. thanks!